siteimprove/admin/class-siteimprove-admin.php

```diff
@@ -163,8 +163,17 @@ public function siteimprove_init() {
 	 * @return void
 	 */
 	private function siteimprove_add_js( $url, $type ) {
+		$file_name = get_option( 'siteimprove_overlayjs_file', 'overlay.js' );
+		$pattern = '/^[a-zA-Z_\d-]+.js/gm';
+
+		if ( preg_match( $pattern, $file_name ) ) {
+			$adjusted_name = Siteimprove::JS_LIBRARY_URL . $file_name;
+		} else {
+			$adjusted_name = $file_name;
+		}
+
```

Comment:
I am not a fan of the variable name `$adjusted_name` because the if else statement means that is a scenario in which `$adjusted_name` is just parsed `$file_name`. 

I would suggest a name like `$smallbox_path` or `$smallbox_script` and then not add in anything in regards "adjusted". 

```suggestion
		$file_name = get_option( 'siteimprove_overlayjs_file', 'overlay.js' );
		$pattern = '/^[a-zA-Z_\d-]+.js/gm';

		if ( preg_match( $pattern, $file_name ) ) {
			$smallbox_path = Siteimprove::JS_LIBRARY_URL . $file_name;
		} else {
			$smallbox_path = $file_name;
		}
```

And then: 
`wp_enqueue_script( 'siteimprove_overlay', $smallbox_path, array(), $this->version, true );`

Reply:
I agree with @jespernpedersen that the name could be better. Just maybe we should call it "overlay" here and not "Smallbox". I know we call it "Smallbox" internally, but this code will be visible publicly and we call it "overlay" there.

Reply:
Done!